### DISCUSSION TITLE
Can I get the same column from DB to excel multiple times ?

### DISCUSSION THREAD

**Discussion by Alsanea-Ala** [pxlrbt/filament-excel#169] [Q&A] [answered]
![Screenshot_20240103_175007](https://github.com/pxlrbt/filament-excel/assets/88606691/d9db6cdd-60f1-4313-9ee7-94dbf96904ac)

but it seem it dose not work

**Comment by pxlrbt** [comment, marked as answer]
Yes, but they need a unique name:

```php
Column::make('name_1')->getStateUsing(fn ($record) => $record->name)->heading('Teacher'),
Column::make('name_2')->getStateUsing(fn ($record) => $record->name)->heading('Teacher'),
Column::make('name_3')->getStateUsing(fn ($record) => $record->name)->heading('Teacher'),
```

**Reply by Alsanea-Ala** [reply]
thx it works

so who getStateUsing() works ??
and the name_3, name_2, and name_1 in make() method is like identifiers for the excel or what ??

**Reply by pxlrbt** [reply]
`getStateUsing()` does the same thing as in Filament. Use the method to get the value for your row.

Yes, the name in `::make()` works as a unique identifier for that column.
